### DISCUSSION TITLE
Quiz App with CustomTkinter

### DISCUSSION THREAD

**Discussion by SarthakTools** [TomSchimansky/CustomTkinter#1901] [Showcase]
### Here is the source code : https://github.com/SarthakTools/Quiz-App

**Comment by SarthakTools** [comment]
Hello akaspace,
Thank you for liking my project but i want to learn about types module and how to implement it in tkinter and customtkinter for changing it window styles???
I want to make a project with you... Project : - Modern Text editor like vscode in customtkinter...

**Reply by Akascape** [reply]
@SarthakTools 
Here is an example program given for changing themes using pywinstyles. https://github.com/Akascape/py-window-styles/blob/main/test_example.py
But I recommend you to implement a 'on-restart theme change` feature if you want to dynamically change theme within the application.

> I want to make a project with you.

I will be really busy in the next months, so I am unable to collab right now or make any new projects. But don't worry, there are many great libraries which will help you.

**Comment by avalon60** [comment]
@avalon60
avalon60
Aug 13, 2023

Cool!

Have you considered an option, offering to list the wrongly answered questions at the end? Showing the wrongly selected and correct answers.
You could also make it shuffle the presented options.

**Reply by SarthakTools** [reply]
### Please you do it and give me the code i will update my code if i find your idea interesting.
